core-libs/setup/ssr/optimized-engine/get-loggable-ssr-optimization-options.ts

```diff
@@ -18,7 +18,10 @@ export const getLoggableSsrOptimizationOptions = (
 ) => {
   const newValue: Record<string, any> = { ...value };
   Object.keys(value).forEach((key) => {
-    if (typeof newValue[key] === 'object') {
+    if (
+      typeof newValue[key] === 'object' &&
+      newValue[key].constructor !== Object
```

Comment:
[NITPICK]

This condition is not obvious to read the intention at the first sight. For readability, can we either add a code comment (`// if it's a class instance) or (IMHO better) extract this condition to a separate local function in this file:
```ts
/**
 * Checks if the given value is a class instance,
 * but not a plain Object.
 */
isClassInstance(value: any): boolean {
  return typeof value === 'object' &&
       value?.constructor !== Object;
}
```